data/hai ships.txt

```diff
@@ -94,6 +161,135 @@ ship "Flea"
 
 
 
+ship "Geocoris"
+	sprite "ship/hai geocoris"
+	attributes
+		category "Heavy Freighter"
+		"cost" 12540000
+		"shields" 13300
+		"hull" 6200
+		"required crew" 15
+		"bunks" 56
+		"mass" 810
+		"drag" 17.1
+		"heat dissipation" 0.45
+		"fuel capacity" 800
+		"cargo space" 761
+		"outfit space" 582
+		"weapon capacity" 172
+		"engine capacity" 94
+		Weapon:
+			"blast radius" 300
+			"shield damage" 3000
+			"hull damage" 1500
+			"hit force" 4500
+	outfits
+		"Pulse Cannon" 2
+		"Pulse Turret" 2
+		"Bullfrog Anti-Missile" 5
+		"Geode Reactor" 2
+		"Hai Valley Batteries"
+		"Hai Diamond Regenerator"
+		"Hai Williwaw Cooling" 5
+		`"Biroo" Atomic Thruster`
+		`"Bondir" Atomic Steering`
+		Hyperdrive
+	engine -88 35
+	engine 88 35
+	gun 22 -104 "Pulse Cannon"
+	gun -22 -104 "Pulse Cannon"
+	turret 16 -19 "Pulse Turret"
+	turret -16 -19 "Pulse Turret"
+	turret 101 -19 "Bullfrog Anti-Missile"
+	turret -101 -19 "Bullfrog Anti-Missile"
+	turret 31 82 "Bullfrog Anti-Missile"
+	turret -31 82 "Bullfrog Anti-Missile"
+	turret 0 -79 "Bullfrog Anti-Missile"
+	explode "tiny explosion" 50
+	explode "small explosion" 50
+	explode "medium explosion" 50
+	explode "large explosion" 50
+	explode "huge explosion" 20
+	"final explode" "final explosion large"
+	description "The introduction of humans to Hai space caused quite a stir among Hai society, especially in the economy where large human freighters were out competing the smaller native Hai freighters. In order to prevent humans from taking over large parts of the Hai economy, the Geocoris was developed, a heavy freighter that surpassed anything humanity had to offer at the time."
```

Comment:
How about this instead?

description "The introduction of Humans caused quite a stir in Hai society, especially in the economy where large human freighters began to out-compete traditionally smaller Hai ships. The Hai responded by developing the Geocoris: a heavy freighter that surpassed anything Humanity had to offer at the time."